Formula/vsftpd.rb

```diff
@@ -30,6 +30,14 @@ class Vsftpd < Formula
     sha256 "c158fac428e06e16219e332c3897c3f730586e55d0ef3a670ed3c716e3de5371"
   end
 
+  # Patch to disable all calls to setrlimit, as macOS, starting from
+  # Monterey does not support this syscall. (reported to developer via
+  # GitHub)
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/e4dd5d67152bb2541c5e38e8bb834ed5b165fcaa/vsftpd/3.0.5.patch"
+    sha256 "95afc3bc00dd6cc37a2c64b19b1e7e30951ec022f839dbab1773b7716966b9cf"
+  end
+
```

Comment:
Shouldn't we apply this patch only on Monterey or newer then?

Reply:
This started failing on Monterey, so yes, if Homebrew provides some ways to apply a patch only for certain platforms and upwards, that could make more sense.

Though, I don't see any reason why this would break Big Sur and below, as the patch disables the ulimit changing code, which was already disabled on eg. OpenBSD given the existing conditional compilation flags that existed in the code.

Reply:
Perhaps not, but the existing code was already working on Big Sur and earlier, so why change it before upstream have?